src/main/webapp/app/localci/build-queue/build-queue.component.ts

```diff
@@ -640,4 +642,13 @@ export class BuildQueueComponent implements OnInit, OnDestroy {
             this.getBuildJobStatistics(span);
         }
     }
+
+    /**
+     * Returns the unique identifier for items in the collection
+     * @param index of a finished buildjob in the collection
+     * @param item current buildjob
+     */
+    trackIdentity(index: number, item: FinishedBuildJob) {
```

Comment:
Do we handle the case where -1 can be returned?

Reply:
This method isn't being used actually, forgot to delete it